k8s/upgrades/0.6.0-0.7.0/jiva-target-patch.tpl.json

```diff
@@ -0,0 +1,84 @@
+{
+  "metadata": {
+     "annotations": {
+       "openebs.io/fs-type": "ext4",
```

Comment:
Is there a possibility of this being XFS ?

Reply:
yes. but this is only used for display and will not impact the volume -- since PV will already be created with the correct mount point.